src/app/shared/components/add-external-cluster-dialog/style.scss

```diff
@@ -28,3 +28,7 @@ mat-dialog-actions {
     margin: unset;
   }
 }
+
+km-button {
```

Comment:
This is very generic. Please add a class on that button and use that as a selector.